### ISSUE TITLE
Prisoners should head to prison when idle

### ISSUE THREAD

**Issue by miki151** [miki151/keeperrl#1504]

**Comment by SoftMonster** [comment]
I agree. If being idle is their only task, you could just leave them in jail there and forget about them :)

**Comment by SoftMonster** [comment]
I mean if you untick all the prisoner's tasks except "Idle", they could be sent back to prison until you want them to come out again.
